sdf_timing/sdfparse.py

```diff
@@ -5,11 +5,24 @@
 from . import sdfwrite
 
 
+def init():
+    sdfyacc.timings = dict()
```

Comment:
Maybe we want to optionally use an [`OrderedDict`](https://docs.python.org/3/library/collections.html#collections.OrderedDict) or [sorteddict](http://www.grantjenks.com/docs/sortedcontainers/sorteddict.html)?